samples/apps/copilot-chat-app/webapp/src/redux/features/users/UsersState.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft. All rights reserved.
+
+import { Constants } from '../../../Constants';
+
+export interface UsersState {
+    users: Users;
+}
+
+export const initialState: UsersState = {
+    users: {
+        bot: Constants.bot.profile,
+    },
+};
+
+export type Users = Record<string, UserData>;
```

Comment:
from previous PR: this should be `UserData | undefined` if some IDs won't have populated objects during loading